modules/buildslave/manifests/install.pp

```diff
@@ -18,6 +21,14 @@
 
     $py_require = Class['packages::mozilla::python27']
 
+    $virtualenv_require = $::operatingsystem ? {
+        windows => [Class['packages::mozilla::python27']],
```

Comment:
similar to comment above, should we define CentOS as the exception and windows, ubuntu, etc the `default`?